### ISSUE TITLE
nice to have: just to click on followers and following on any user

### ISSUE THREAD

**Issue by dirmansebastian** [AkashaProject/dapp#99]
![click_followers](https://user-images.githubusercontent.com/15945292/34695981-995de61e-f4d5-11e7-8ec3-25a07a76cff2.JPG)

**Comment by bogdanbrezuica** [comment]
What do you have in mind? A modal with the list of followers/following, similar to the one you see when clicking on an entry's score?

**Comment by dirmansebastian** [comment]
yup, looks good, similar to the one when i click on an entry's score

**Comment by bogdanbrezuica** [comment]
Nice suggestion, thanks :)
